packages/kotti-dropdown-button/KtDropdownButton.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+	<div class="kt-dropdown-button">
+		<KtPopover v-bind="$attrs" :placement="placement">
+			<KtButtonGroup>
+				<KtButton class="kt-dropdown-button__button" v-bind="$attrs">
+					<slot>Dropdown Text</slot>
+				</KtButton>
+				<KtButton
+					class="kt-dropdown-button__icon"
+					v-bind="$attrs"
+					:icon="dropdownIcon"
+				>
+					<i class="yoco" v-text="dropdownIcon" />
+				</KtButton>
+			</KtButtonGroup>
+			<div slot="content">
+				<slot name="KtDropdownMenus">Dropdown Menu</slot>
+			</div>
+		</KtPopover>
+	</div>
+</template>
+<script>
+import KtButton from '../kotti-button'
+import KtButtonGroup from '../kotti-button-group'
+import KtPopover from '../kotti-popover'
+
+export default {
+	name: 'KtDropdownButton',
+	props: {
+		dropdownIcon: {
+			type: String,
+			default: 'chevron_up',
```

Comment:
I think most dropdown default to chevrin_down not up
also opening the drop down down
it's very strange to see the arrow pointing up
the only exception is if the dropdown is in the bottom of the ui at which point maybe like popover we could have the icon default to poointing at the side the popover would open, unless it was explicitly set

Reply:
Yes I kind of did that thinking of bottom buttons. I don't mind defaulting down.